ecc_enclave/enclave/shim.cpp

```diff
@@ -64,6 +68,8 @@ void get_state(
     COND2LOGERR(encoded_cipher_len != encoded_cipher_s.size(), "Unexpected string length");
 
     // base64 decode
+    // TODO: double check if Fabric handles values as strings; if it handles binary values, remove
```

Comment:
fabric has only byte version of getstate/putstate in [shim](https://pkg.go.dev/github.com/hyperledger/fabric-chaincode-go/shim). We might have though some debug statement which might rely on it being a string

btw: i notice now that  also documents a `DelState` function.  Can't recall have seen that in the past but matches the `deleted` flag from the protobuf.

Reply:
but why would we convert these to string? I had noticed that, and it's unnecessary.